main.py

```diff
@@ -87,6 +87,12 @@ def servenow(netpdata="127.0.0.1", chatport="9696"):
         print_formatted_text(HTML("[" + obtntime() + "] " + "<b>SNCTRYZERO</b> > <red><b>SNCTRYZERO server was shut down</b></red>"))
         sys.exit()
 
+        
+def getipaddr():
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))
+    return s.getsockname()[0]
```

Comment:
The IP address provided here would be restricted to only IPv4 addresses.

Take a look at this.

![image](https://user-images.githubusercontent.com/49605954/96951685-a5c5ec00-150a-11eb-95ea-6a832e485a76.png)

Even when IPv6 is selected, only the IPv4 address is displayed.

Also, the displayed IP address is not very useful when communicating across dissimilar networks as the address is applicable only in the intranet.

![image](https://user-images.githubusercontent.com/49605954/96951569-68615e80-150a-11eb-9f42-185b0f728530.png)